cohort-engine/src/main/java/com/ibm/cohort/engine/LoggingEnum.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *
+ *  * (C) Copyright IBM Corp. 2021
+ *  *
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package com.ibm.cohort.engine;
+
+public enum LoggingEnum {
+	COVERAGE("coverage"),
+	TRACE("trace"),
+	NA(null);
+
+	private String debugLevel;
```

Comment:
I'm a little confused on the purpose of the `debugLevel` variable.
All interactions with the enum directly refer to the uppercase enum value itself (including `getLoggingFromString()` below).
I also can't find any uses of `getDebugLevel()` being used in the code.

Unless it serves some deeper purpose I'm missing, I'd say remove the variable entirely.

Reply:
Yeah, I thought I needed it, got distracted, and then never refactored the class. Whoops >.< but this and your other comment re the enum are both fixed